src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -170,6 +170,14 @@ internal void Rehydrate(string serializedInput)
                     this.context.Name,
                     this.context.InstanceId,
                     isReplay: false));
+
+                this.Config.TraceHelper.OrchestratorStateChange(
+                    this.context.HubName,
+                    this.context.Name,
+                    this.context.InstanceId,
+                    FunctionType.Orchestrator,
+                    FunctionState.Failed,
```

Comment:
Why is this considered a failure? I'm not super familiar with the entity code, but line 168 gives the impression that the entity is actually being *created*.

Reply:
Relatedly - should this have `FunctionType.Entity` as a parameter instead?